tools/copyAccessLogFileTool.php

```diff
@@ -175,19 +175,41 @@ function _copyFile($filePath) {
 		}
 
 		// Uncompress it, if needed.
-		$gunzipPath = escapeshellarg(Config::getVar('cli', 'gunzip'));
+		$gunzipPath = Config::getVar('cli', 'gunzip');
 		if ($isCompressed) {
-			exec($gunzipPath . ' ' . $tmpFilePath);
+			if (!is_executable($egrepPath)) {
+				printf(__('admin.copyAccessLogFileTool.error.executingUtil', array('utilPath' => $gunzipPath, 'utilVar' => 'gunzip')) . "\n");
+				exit(1);
+			}
+			$gunzipPath = escapeshellarg($gunzipPath);
+			$output = array();
+			$returnValue = 0;
+			exec($gunzipPath . ' ' . $tmpFilePath, $output, $returnValue);
+			if ($returnValue > 1) {
```

Comment:
guzip returns 1 when an error occurs and 2 in case of warning, I think that this check will only get the warning one.